bin/console

```diff
@@ -1,11 +1,25 @@
 #!/usr/bin/env ruby
```

Comment:
This file is structured to allow `bin/console` to open an IRB console with the test environment wired up, which has a SQLite database and ActiveRecord models, as well as having the ActiveSet code loaded. This makes development and exploration much easier.